src/chart_types/xy_chart/renderer/canvas/xy_chart.tsx

```diff
@@ -178,11 +184,19 @@ class XYChartComponent extends React.Component<XYChartProps> {
           }}
           // eslint-disable-next-line jsx-a11y/no-interactive-element-to-noninteractive-role
           role="presentation"
+          aria-describedby="get_custom_description"
```

Comment:
You'll want to replace this with a randomly generated `id` – if a user renders two charts, each with a unique description, you'll end up getting only the first `id` read off for both charts.

You'll also want to conditionally add the attribute. An empty description will sometimes still get read out (something like "description empty") but probably just want to ignore it.

```suggestion
		  {...(customDescription ? {'aria-describedby': randomId} : {})}
```

Reply:
you can use the `chartId`, that is random and unique per chart, + some postfix

Reply:
Ok I am open to feedback about 3b13cc9 thank you!